nodejs/examples/influxdb.js

```diff
@@ -1,25 +1,53 @@
-const influxdb = require('influxdb-nodejs');
+const Influx = require("influx");
 const config = require("platformsh-config").config();
 
-exports.usageExample = async function() {
-
-    const credentials = config.credentials('influxdb');
-
-    // Connecting to the InfluxDB server. By default it has no user defined, so you will need to create it.
-    let connectionString = `http://${credentials.host}:${credentials.port}/`;
-    let client = new influxdb(connectionString);
-
-    let password = Math.random().toString(36).slice(-8);
-    let response = await client.queryPost(`create user "deploy_user" with password \'${password}\' with all privileges`);
-
-    // Now reconnect with an authenticated connection so that we can access a database.
-    connectionString = `http://deploy_user:${password}@${credentials.host}:${credentials.port}/deploys`;
-    client = new influxdb(connectionString);
-
-    await client.createDatabase();
-
-    // And remove the user.
-    response = await client.queryPost(`delete user "deploy_user"`);
-
-    return output;
+exports.usageExample = async function run() {
```

Comment:
The elasticsearch example file doesn't name this function. Why is it named here?

Reply:
Woops. I was trying things out so I modified this file to run it separately from the rest and forgot I named this function when assigning it back to `exports.usageExample`.